src/w2form.js

```diff
@@ -1254,9 +1254,7 @@
             var url = (typeof this.url != 'object' ? this.url : this.url.get);
             if (url && this.recid != 0 && this.recid != null) {
                 this.request();
-            } else {
-                this.refresh();
-            }
+            } 
```

Comment:
I do not understand why you changed it. If URL is defined, then refresh from URL, otherwise, refresh from memory. This change will cause problems if your form is local